### PR TITLE
Add script and ressource file to deploy librephotos on debian 11

### PR DESCRIPTION
**Pull request by tomamplius**
Thank for your review

### CODE REVIEW

File: install-librephotos.sh [hunk 1]
```diff
@@ -0,0 +1,73 @@
+#!/usr/bin/env bash
+set -exa
+
+# PRE INSTALL
+grep librephotos /etc/group > /dev/null || groupadd -r librephotos
```
Comment (fabienwang) [inline install-librephotos.sh:5]: I 'd have used:
```
id -g librephotos > /dev/null || groupadd -r librephotos
```

same for all groups

Comment (tomamplius) [reply install-librephotos.sh]: done
merci

File: README.md [hunk 2]
```diff
@@ -0,0 +1,91 @@
...
+~~~
+cd /tmp/
+git clone git@github.com:LibrePhotos/librephotos-linux.git
+cd librephotos-linux
+./install-librephotos.sh 
+~~~
+
+Other way can be use download archive
+~~~
+wget https://github.com/librephotos/librephotos-linux/archive/main.zip -O /tmp/main.zip
```
Comment (tomamplius) [inline README.md:22]: This will work after commit. use https://github.com/tomamplius/librephotos-linux/archive/main.zip before

File: README.md [hunk 3]
```diff
@@ -0,0 +1,91 @@
...
+  - redis
+
+## Installation
+
+### Debian like distribution
+
+Execute following commande as root
+~~~
+cd /tmp/
+git clone git@github.com:LibrePhotos/librephotos-linux.git
```
Comment (tomamplius) [inline README.md:15]: use "git clone git@github.com:tomamplius/librephotos-linux.git" before merge

**Comment by derneuere** [comment]
Thanks again for your pull request! I merge these changes now too, because I merged the other branch.

**Pull request merged**